jupyterlab_caip_optimizer/src/components/dashboard.tsx

```diff
@@ -0,0 +1,209 @@
+    <ChevronLeft {...props} ref={ref} />
+  )),
+  ResetSearch: forwardRef((props, ref: React.Ref<SVGSVGElement>) => (
+    <Clear {...props} ref={ref} />
+  )),
+  Search: forwardRef((props, ref: React.Ref<SVGSVGElement>) => (
+    <Search {...props} ref={ref} />
+  )),
+  SortArrow: forwardRef((props, ref: React.Ref<SVGSVGElement>) => (
+    <ArrowUpward {...props} ref={ref} />
+  )),
+  ThirdStateCheck: forwardRef((props, ref: React.Ref<SVGSVGElement>) => (
+    <Remove {...props} ref={ref} />
+  )),
+  ViewColumn: forwardRef((props, ref: React.Ref<SVGSVGElement>) => (
+    <ViewColumn {...props} ref={ref} />
+  )),
+};
+
+const dateFormat = 'h:mm a, MMM. D, YYYY';
+const columns = [
+  {
+    title: 'Name',
+    field: 'name',
+    render: study => prettifyStudyName(study.name),
+  },
+  { title: 'State', field: 'state' },
+  {
+    title: 'Objective',
+    field: 'metrics',
+  },
+  {
+    title: 'Date Created',
+    field: 'createTime',
+    render: study => moment(study.createTime).format(dateFormat),
+    customSort: (a, b) => {
+      const aDate = moment(a.createTime);
+      const bDate = moment(b.createTime);
+      return aDate.diff(bDate);
+    },
+  },
+];
+
+function capitalizeFirstLetter(string: string): string {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+interface Props {
+  loading: boolean;
+  error?: string;
+  studies?: Study[];
+  openCreateStudy: () => void;
+  openStudyDetails: (studyName: string) => void;
+  deleteStudy: (studyName: string) => void;
+}
+
+interface MappedStudy {
+  name: Study['name'];
+  createTime: Study['createTime'];
+  state: Study['state'];
+  metrics: string;
+}
+
+const mapStateToProps = state => ({
+  loading: state.studies.loading,
+  error: state.studies.error,
+  studies: state.studies.data,
+});
+
+const mapDispatchToProps = dispatch => ({
+  openCreateStudy: () => dispatch(setView({ view: 'createStudy' })),
+  openStudyDetails: (studyName: string) =>
+    dispatch(setView({ view: 'studyDetails', studyId: studyName })),
+  deleteStudy: (studyName: string) => dispatch(deleteStudy(studyName)),
+});
+
+export const DashboardUnwrapped: React.FC<Props> = ({
+  loading,
+  error,
+  studies,
+  openCreateStudy,
+  openStudyDetails,
+  deleteStudy,
+}) => {
+  // studies needed to be mapped since material table edits props (a very bad antipattern) and redux (using immer) freezes the data
+  // thus the data needs to be copied to become mutable
+  // read more here:
+  // https://stackoverflow.com/questions/59648434/material-table-typeerror-cannot-add-property-tabledata-object-is-not-extensibl
+  const mappedStudies: MappedStudy[] = studies
+    ? studies.map(study => ({
+        name: study.name,
+        createTime: study.createTime,
+        state: study.state,
+        metrics: study.studyConfig.metrics
+          .map(
+            metric =>
+              `${capitalizeFirstLetter(metric.goal.toLowerCase())} "${
```

Comment:
I would move the toLowerCase() into the helper function and rename it something like "makeReadable"